web/src/components/card.js

```diff
@@ -21,14 +21,14 @@ export default function Card({
         : 'bg-white text-brand-1';
     const transparentStyle = isTransparent ? 'bg-transparent text-black' : '';
 
-    const cardStyle = `rounded-bl-lg w-11/12 md:w-7/12 pl-6
+    const cardStyle = `pl-6 rounded-bl-4xl w-11/12 md:w-6/12 
    ${darkStyle} ${transparentStyle}
     ${isRight ? 'md:transform md:translate-x-3/4' : ''}`;
```

Comment:
I didn't catch this before but I don't think this is how we want to handle this. Using a combination of `margin-right` and `margin-left` gets the correct alignment.

`${isRight ? 'md:ml-auto' : 'md:mr-auto'}`;

Reply:
@asasmith thats not it.

Reply:
@kcbreuil what's not it?

Reply:
ml-auto does not exist

Reply:
![Screen Shot 2021-06-10 at 1 53 34 PM](https://user-images.githubusercontent.com/11489171/121573429-43672080-c9f3-11eb-9eda-2275537beb9e.png)

Reply:
@asasmith Thats not the logic we need--i am not positioning this with margin?

Reply:
you can see that in the image above